### PR TITLE
Feature/job progress extras

### PR DESCRIPTION
**Pull request by alexwelch**

### CODE REVIEW

File: app/models/step.rb [hunk 1]
```diff
@@ -6,8 +6,10 @@ class Step < BaseResource
     string :source
   end
 
-  def get_status(steps_completed)
-    if (steps_completed.to_i + 1) == order
+  def get_status(steps_completed, status)
```
Comment (dpetersen) [inline app/models/step.rb]: Should there be constants for any of this stuff? `error` and `complete` show up in a couple of places.

**Comment by dpetersen** [comment]
:+1: :cat2:

File: app/assets/stylesheets/panamax/deployments.css.scss [hunk 2]
```diff
@@ -494,16 +520,30 @@
         text-decoration: underline;
       }
 
-      > .complete h5 > .status-info {
-        background-image: image-url('icon_status_running.png');
+      > .complete {
```
Comment (dpetersen) [inline app/assets/stylesheets/panamax/deployments.css.scss]: Good use of the cascade here. :clap:

File: app/views/deployment_targets/index.html.haml [hunk 3]
```diff
@@ -11,11 +11,13 @@
 
 - @jobs.each do |job|
   - presenter = JobPresenter.new(job, self)
-  .deployment-job{ id: "#{presenter.dom_id}" }
-    .deployment-job-progress{ class: "intially-#{presenter.status}", data: { source: job_path(job.key) } }
+  .deployment-job{ class: "intially-#{presenter.status}", id: "#{presenter.dom_id}" }
```
Comment (dpetersen) [inline app/views/deployment_targets/index.html.haml]: initially spelling

**Comment by argvader** [comment]
:+1: its been so long since i've been in the ui codebase i may have lost my plus 1 creds

**Comment by dpetersen** [comment]
We'll take it!

**Pull request merged**